slangpy/tests/device/test_standard_modules.py

```
@@ -0,0 +1,192 @@
+            usage=spy.BufferUsage.shader_resource | spy.BufferUsage.unordered_access,
+        )
+
+        kernel.dispatch(thread_count=[1, 1, 1], result=result)
+
+        np.testing.assert_allclose(result.to_numpy().view(dtype=np.float32), [2.0])
+
+    finally:
+        device.close()
+
+
+@pytest.mark.parametrize("target,extension", [("hlsl", "hlsl"), ("metal", "metal")])
```

Ruff (0.15.18)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)